packages/fxa-settings/src/models/reliers/options/oauth-relier-options.ts

```diff
@@ -0,0 +1,42 @@
+import { RelierOptions } from "./relier-options";
+
+export interface OAuthClientInfo {
+  clientId: string;
+  imageUri: string | undefined;
+  serviceName: string;
+  // This can be a single uri or comma separated list
+  redirectUri: string | undefined;
+  trusted: boolean;
+}
+
+export interface OAuthVerification {
+  accessType: string | undefined;
+  acrValues: string | undefined;
+  action: string;
```

Comment:
I tried to figure which parameters were required and which ones were optional. The types should reflect this.